expressServer/scripts/deleteAllPasswords.js

```diff
@@ -0,0 +1,39 @@
+const dotenv = require('dotenv');
+const { connectDB, closeDB } = require('../database/database.js');
+const { User } = require('../database/models/User.js');
+
+dotenv.config();
+
+async function encryptAllUserPasswords() {
+    const batchSize = 1000;
+    let totalProcessed = 0;
+    let skip = 0;
+
+    try {
```

Comment:
You can remove the try

Reply:
The then catch and finally sequences ask for the try method, should I remove it anyway?